docs/partners/2-certification/certification-for-data-providers/readme.md

```diff
@@ -14,6 +14,18 @@ across all Ed-Fi Alliance certifications.
 Specific certifications describe additional requirements; those can be found
 in [Available Certifications](../available-certifications/readme.md).
 
+## Alignment with new Break-Rest Development Cadence
+
+![Break-Rest Development Cadence Table](https://edfi.atlassian.net/wiki/download/attachments/20876154/image-2024-3-30_10-49-43.png?version=1&modificationDate=1711813783893&cacheVersion=1&api=v2 "Break Rest Development Cadence")
+
+Certification requirements are aligned to the Ed-Fi Data and API Standard Release Cadence
+* Support goes from 3 to 4 years
```

Comment:
Extra line break above the bullets makes this more readable in plain text.

```suggestion

* Support goes from 3 to 4 years
```

Should there be a period at the end of each bullet point?

Reply:
I'll follow the practice you've established already